pkg/server/server.go

```diff
@@ -727,6 +742,88 @@ func createDeployment(w http.ResponseWriter, r *http.Request) {
 	helper.LogInfo.Printf("Created Deployment: %s\n", dep.GetName())
 }
 
+func getConfigs(w http.ResponseWriter, r *http.Request) {
+	pathVars := mux.Vars(r)
+
+	if os.Getenv("DEPLOY_STATE") == "PROD" {
```

Comment:
This is more of an enhancement but it would be slick to start using [middleware](https://github.com/justinas/alice) and putting this into a middleware that all handlers will run.